src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -289,46 +270,7 @@ End Class
             End Using
         End Function
 
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function TestDown1() As Task
```

Comment:
💭 Where has this gone to?

Reply:
Covered by the editor in the new implementation.

---
In reply to: [208003746](https://github.com/dotnet/roslyn/pull/29016#discussion_r208003746) [](ancestors = 208003746)